pootle/templates/browser/overview.html

```diff
@@ -75,10 +75,18 @@ <h4 class="clear">{{ item.title }}</h4>
 {% endblock %}
 
 {% block pre_content %}
-{% if announcement %}
+{% if announcement or upload_form or display_download %}
```

Comment:
Isn't this the case where we want to be using overload - or whatever its called?